measures/2020/measures-data.json

```diff
@@ -9517,5 +9517,345 @@
     ],
     "measureSets": [],
     "measureSpecification": {}
+  },
+  {
+    "category": "cost",
+    "title": "Medicare Spending Per Beneficiary (MSPB)",
+    "description": "The risk-adjusted cost to Medicare for all Parts A and B services performed for an inpatient beneficiary as a result of a clinician’s care during the period 3 days prior to the patient’s hospital stay through 30 days after discharge.",
+    "measureId": "MSPB_1",
+    "metricType": "costScore",
+    "firstPerformanceYear": 2017,
+    "lastPerformanceYear": null,
+    "isInverse": true,
+    "overallAlgorithm": "simpleAverage",
+    "submissionMethods": [
+      "administrativeClaims"
+    ],
+    "measureSpecification": {
+      "measureInformation": "https://qpp.cms.gov/docs/cost_specifications/2019-12-17-mif-tpcc.pdf"
```

Comment:
One change.  We determined we were going to use `default`, not `measureInformation`.

Reply:
For cost measures? We used `default` for `PI` in #315.

Reply:
Yes.  As a pattern, if the measure spec is not associated with a `submissionMethod` (which `measureInformation` is not), then the key will be `default`

Reply:
@joshboley addressed feedback.